DESCRIPTION

```diff
@@ -27,8 +27,7 @@ Imports:
     rstudioapi,
     shiny
 Suggests: 
-    commonmark,
-    DT,
+    gt,
```

Comment:
👀

Reply:
Did I introduce a buglet here that I'm not noticing? Or you're just observing the change?😄

Reply:
Still trying to make the "Copy to clipboard" change happen, and it seems like gt may be a better candidate as the tool to do so.

Reply:
Ah, sorry, that was not a critical side eye, that was just me trying to say "you made it happen!" 😆